typeorm-seeding.config.ts

```diff
@@ -0,0 +1,17 @@
+const typeormForSeedConfig = {
```

Comment:
I suggest adding a factory to build base config for packages `typeorm-seeding` and `@nestjs/typeorm` to have consistency.

Reply:
Resolved on ba9b0723ef9f71d8cb90f1ff7c59efc550423222.